ci-operator/config/openshift/verification-tests/openshift-verification-tests-master__ota-nightly-4.14.yaml

```diff
@@ -0,0 +1,91 @@
+base_images:
+  aws-efs-csi-operator-create-efs:
+    name: "4.14"
+    namespace: ocp
+    tag: aws-efs-csi-operator-create-efs
+  cli:
+    name: "4.14"
+    namespace: ocp
+    tag: cli
+  dev-scripts:
+    name: test
+    namespace: ocp-kni
+    tag: dev-scripts
+  tests-private:
+    name: tests-private
+    namespace: ci
+    tag: "4.14"
+  tests-private-ui:
+    name: tests-private-ui
+    namespace: ci
+    tag: "4.14"
+  tools:
+    name: "4.14"
+    namespace: ocp
+    tag: tools
+  upi-installer:
+    name: "4.14"
+    namespace: ocp
+    tag: upi-installer
+build_root:
+  image_stream_tag:
+    name: release
+    namespace: openshift
+    tag: golang-1.18
+releases:
+  latest:
+    candidate:
+      architecture: amd64
+      product: ocp
+      stream: nightly
+      version: "4.14"
+resources:
+  '*':
+    limits:
+      memory: 8Gi
+    requests:
+      cpu: "1"
+      memory: 4Gi
+tests:
+- as: aws-ipi-disc-priv-f28-longduration
+  cron: 23 20 19 * *
+  steps:
+    cluster_profile: aws-qe
+    env:
+      BASE_DOMAIN: qe.devcluster.openshift.com
+      E2E_RUN_TAGS: '@ota'
+      EXTRACT_MANIFEST_INCLUDED: "true"
+      TEST_FILTERS: DisconnectedOnly&
+      TEST_SCENARIOS: OTA
+    test:
+    - chain: openshift-e2e-test-qe-longduration
+    workflow: cucushift-installer-rehearse-aws-ipi-disconnected-private
+- as: azure-ipi-f28
+  cron: 33 3 19 * *
+  steps:
+    cluster_profile: azure-qe
+    env:
+      BASE_DOMAIN: qe.azure.devcluster.openshift.com
+      E2E_RUN_TAGS: '@ota'
+      TEST_FILTERS: ~DisconnectedOnly&;~Disruptive&
+      TEST_SCENARIOS: OTA
+    test:
+    - chain: openshift-e2e-test-qe
+    workflow: cucushift-installer-rehearse-azure-ipi
+- as: gcp-ipi-ovn-ipsec-f28-destructive
```

Comment:
The OTA case can not be executed in the regular ci as what other teams did ?

Reply:
This part will be executed in regular ci together with other cases, but sometimes we need to get all ota cases test result in one job and one day(such as full-functional test, especially osus test) , so I add one more dedicated job together with above missing one in verification-tests repo for the purpose.

Reply:
Generally in regular ci, there are a lot of destructive ci jobs, we should be able to collect test results quickly from the latest test run. If we can collect the results from there, I would still recommended to run them in regular ci.

Reply:
From past experience, whether destructive cases or other cases which should be executed in regular ci, there are following situation:
1. we need get latest test result(should be in the last one/two weeks), but no test results for all kinds of reasons(test frequency/some other filters/flakes)
2. then we have to re-trigger several jobs to get the test result(because the cases are split with tags in different jobs)
3. in the re-triggered jobs, all other teams' cases also will be executed, that's a long waiting time.

so my suggestion is to add one more dedicated jobs including ota cases only for such purpose, and at the same time, those cases will also be executed in regular ci with other team cases for platform coverage.

Reply:
If that, we can add dedicated OTA ci jobs in regular ci to save introducing new ci yaml files. Especially for OSUS test cases execution results, if we can have specific OSUS tag for the cases, that would be better, that can save us more time.